### DISCUSSION TITLE
What is the difference between "copy" and "copy-equal"?

### DISCUSSION THREAD

**Discussion by 1000Fear** [mpv-player/mpv#14721] [Q&A] [answered]
For `profile-restore=<...>` I know `copy` is used to restore old values, but I don't quite understand the description of `copy-equal` in the reference.

>`copy-equal`
 Similar to `copy`, but restore an option only if it has the same value as the value effectively set by the profile. This tries to deal with the situation when the user does not want the option to be reset after interactively changing it.

The URL for this option: https://mpv.io/manual/master/#runtime-profiles

Sorry, I am not a native English speaker, it's a bit difficult for me to figure it out.
Can you teach me in simple and understandable explanation?

**Comment by llyyr** [comment]
The options set by the profile will only be restored if the current value is equal to the value set by the profile. This is so options you set during runtime aren't undone.

**Comment by verygoodlee** [comment, marked as answer]
In this case, if you manually change the volume from 100 to 80 while pausing, 
`copy` can restore volume from 80 to 50, 
but `copy-equal` can't, because 80 not equals 100.
```ini
volume=50
[test]
profile-cond=pause
profile-restore=copy-equal
volume=100
```

**Reply by 1000Fear** [reply]
Thank you! exactly what I need.
